src/assets/translations/en/main.json

```diff
@@ -2325,6 +2325,7 @@
     "pricePerAsset": "%{from} per %{to}",
     "symAlert": "Your %{from} will be split into a 50:50 exposure to both %{to} and %{from} upon entering the pool.",
     "symAfterRuneAlert": "Symmetrical deposits are not possible following an asymmetric RUNE deposit due to the way the THOR network works.",
+    "withdrawTypeNotAvailable": "This withdraw type is not available for this pool.",
```

Comment:
```suggestion
    "withdrawTypeNotAvailable": "This withdrawal type is not available for this pool.",
```

Reply:
Updated in [b602e31](https://github.com/shapeshift/web/pull/7351/commits/b602e31990567afc87ae66b6a0c081c5b9bbdb26)